include/fmt/format.h

```diff
@@ -1039,10 +1039,10 @@ template <typename T, size_t SIZE, typename Allocator>
 struct is_contiguous<basic_memory_buffer<T, SIZE, Allocator>> : std::true_type {
 };
 
-namespace detail {
+FMT_BEGIN_DETAIL_NAMESPACE
 FMT_API bool write_console(std::FILE* f, string_view text);
 FMT_API void print(std::FILE*, string_view);
-}  // namespace detail
+FMT_END_DETAIL_NAMESPACE
```

Comment:
I think we should use `FMT_{BEGIN,END}_EXPORT` instead of "magically" folding exports into namespace macros.

Reply:
Ok, will do!

Reply:
Done!